pallets/frequency-tx-payment/src/tests/mock.rs

```diff
@@ -252,6 +260,8 @@ impl GetStableWeight<RuntimeCall, Weight> for TestCapacityCalls {
 			RuntimeCall::Balances(BalancesCall::transfer_allow_death { .. }) =>
 				Some(Weight::from_parts(11, 0)),
 			RuntimeCall::Msa(pallet_msa::Call::create { .. }) => Some(Weight::from_parts(12, 0)),
+			RuntimeCall::Msa(pallet_msa::Call::add_public_key_to_msa { .. }) =>
```

Comment:
This is for a more realistic test of the call; otherwise it just errors saying the call isn't a Capacity eligible call.